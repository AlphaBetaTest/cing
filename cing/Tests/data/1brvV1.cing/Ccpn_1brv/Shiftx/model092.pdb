REMARK PDB file of molecule Ccpn_1brv                                 
ATOM      1  N   VAL A 171       5.100   0.151   5.005  1.00  0.00              
ATOM      2  CA  VAL A 171       4.396   0.815   3.882  1.00  0.00              
ATOM      3  HA  VAL A 171       3.851   1.710   4.234  1.00  0.00              
ATOM      4  CB  VAL A 171       5.496   1.390   2.926  1.00  0.00              
ATOM      5  HB  VAL A 171       4.935   1.807   2.090  1.00  0.00              
ATOM      6  CG1 VAL A 171       6.294   2.569   3.538  1.00  0.00              
ATOM      7 HG11 VAL A 171       5.632   3.395   3.856  1.00  0.00              
ATOM      8 HG12 VAL A 171       6.880   2.260   4.424  1.00  0.00              
ATOM      9 HG13 VAL A 171       7.017   3.003   2.824  1.00  0.00              
ATOM     10  CG2 VAL A 171       6.497   0.395   2.294  1.00  0.00              
ATOM     11 HG21 VAL A 171       5.971  -0.377   1.709  1.00  0.00              
ATOM     12 HG22 VAL A 171       7.185   0.894   1.585  1.00  0.00              
ATOM     13 HG23 VAL A 171       7.122  -0.120   3.046  1.00  0.00              
ATOM     14  C   VAL A 171       3.266  -0.101   3.269  1.00  0.00              
ATOM     15  O   VAL A 171       3.487  -0.727   2.229  1.00  0.00              
ATOM     16  N   PRO A 172       2.030  -0.213   3.853  1.00  0.00              
ATOM     17  CD  PRO A 172       1.764   0.109   5.272  1.00  0.00              
ATOM     18  CA  PRO A 172       0.912  -0.991   3.248  1.00  0.00              
ATOM     19  HA  PRO A 172       1.269  -1.698   2.472  1.00  0.00              
ATOM     20  CB  PRO A 172       0.442  -1.809   4.467  1.00  0.00              
ATOM     21  HB2 PRO A 172      -0.641  -2.037   4.443  1.00  0.00              
ATOM     22  HB3 PRO A 172       0.956  -2.790   4.467  1.00  0.00              
ATOM     23  CG  PRO A 172       0.841  -1.027   5.727  1.00  0.00              
ATOM     24  HG2 PRO A 172      -0.042  -0.629   6.257  1.00  0.00              
ATOM     25  HG3 PRO A 172       1.358  -1.688   6.449  1.00  0.00              
ATOM     26  HD2 PRO A 172       1.265   1.094   5.355  1.00  0.00              
ATOM     27  HD3 PRO A 172       2.661   0.149   5.917  1.00  0.00              
ATOM     28  C   PRO A 172      -0.195  -0.064   2.634  1.00  0.00              
ATOM     29  O   PRO A 172      -0.788   0.765   3.336  1.00  0.00              
ATOM     30  N   CYS A 173      -0.521  -0.229   1.338  1.00  0.00              
ATOM     31  H   CYS A 173       0.039  -0.900   0.799  1.00  0.00              
ATOM     32  CA  CYS A 173      -1.552   0.588   0.630  1.00  0.00              
ATOM     33  HA  CYS A 173      -1.255   1.640   0.661  1.00  0.00              
ATOM     34  CB  CYS A 173      -1.452   0.250  -0.857  1.00  0.00              
ATOM     35  HB2 CYS A 173      -0.838  -0.628  -1.067  1.00  0.00              
ATOM     36  HB3 CYS A 173      -2.448   0.092  -1.249  1.00  0.00              
ATOM     37  SG  CYS A 173      -0.601   1.529  -1.773  1.00  0.00              
ATOM     38  C   CYS A 173      -2.998   0.587   1.237  1.00  0.00              
ATOM     39  O   CYS A 173      -3.667   1.625   1.197  1.00  0.00              
ATOM     40  N   SER A 174      -3.448  -0.529   1.846  1.00  0.00              
ATOM     41  H   SER A 174      -2.788  -1.314   1.778  1.00  0.00              
ATOM     42  CA  SER A 174      -4.691  -0.553   2.687  1.00  0.00              
ATOM     43  HA  SER A 174      -5.535  -0.271   2.050  1.00  0.00              
ATOM     44  CB  SER A 174      -4.924  -1.998   3.184  1.00  0.00              
ATOM     45  HB2 SER A 174      -5.872  -2.037   3.759  1.00  0.00              
ATOM     46  HB3 SER A 174      -5.071  -2.678   2.327  1.00  0.00              
ATOM     47  OG  SER A 174      -3.861  -2.477   4.016  1.00  0.00              
ATOM     48  HG  SER A 174      -3.045  -2.389   3.505  1.00  0.00              
ATOM     49  C   SER A 174      -4.788   0.448   3.894  1.00  0.00              
ATOM     50  O   SER A 174      -5.876   0.683   4.420  1.00  0.00              
ATOM     51  N   THR A 175      -3.656   1.025   4.308  1.00  0.00              
ATOM     52  H   THR A 175      -2.834   0.713   3.778  1.00  0.00              
ATOM     53  CA  THR A 175      -3.565   2.036   5.400  1.00  0.00              
ATOM     54  HA  THR A 175      -4.551   2.092   5.901  1.00  0.00              
ATOM     55  CB  THR A 175      -2.556   1.541   6.479  1.00  0.00              
ATOM     56  HB  THR A 175      -2.620   2.232   7.344  1.00  0.00              
ATOM     57  OG1 THR A 175      -1.217   1.576   5.991  1.00  0.00              
ATOM     58  HG1 THR A 175      -1.229   1.202   5.100  1.00  0.00              
ATOM     59  CG2 THR A 175      -2.834   0.114   6.993  1.00  0.00              
ATOM     60 HG21 THR A 175      -3.883   0.008   7.330  1.00  0.00              
ATOM     61 HG22 THR A 175      -2.697  -0.640   6.192  1.00  0.00              
ATOM     62 HG23 THR A 175      -2.171  -0.165   7.825  1.00  0.00              
ATOM     63  C   THR A 175      -3.262   3.499   4.905  1.00  0.00              
ATOM     64  O   THR A 175      -3.548   4.451   5.633  1.00  0.00              
ATOM     65  N   CYS A 176      -2.686   3.702   3.695  1.00  0.00              
ATOM     66  H   CYS A 176      -2.747   2.899   3.064  1.00  0.00              
ATOM     67  CA  CYS A 176      -2.504   5.041   3.078  1.00  0.00              
ATOM     68  HA  CYS A 176      -2.027   5.723   3.810  1.00  0.00              
ATOM     69  CB  CYS A 176      -1.507   4.763   1.931  1.00  0.00              
ATOM     70  HB2 CYS A 176      -1.866   4.063   1.184  1.00  0.00              
ATOM     71  HB3 CYS A 176      -1.308   5.634   1.286  1.00  0.00              
ATOM     72  SG  CYS A 176       0.064   4.190   2.604  1.00  0.00              
ATOM     73  C   CYS A 176      -3.863   5.709   2.658  1.00  0.00              
ATOM     74  O   CYS A 176      -4.047   6.911   2.851  1.00  0.00              
ATOM     75  N   GLU A 177      -4.787   4.922   2.069  1.00  0.00              
ATOM     76  H   GLU A 177      -4.421   4.020   1.759  1.00  0.00              
ATOM     77  CA  GLU A 177      -6.201   5.294   1.783  1.00  0.00              
ATOM     78  HA  GLU A 177      -6.513   4.548   1.040  1.00  0.00              
ATOM     79  CB  GLU A 177      -7.064   5.069   3.066  1.00  0.00              
ATOM     80  HB2 GLU A 177      -6.677   5.725   3.871  1.00  0.00              
ATOM     81  HB3 GLU A 177      -8.093   5.440   2.891  1.00  0.00              
ATOM     82  CG  GLU A 177      -7.173   3.617   3.608  1.00  0.00              
ATOM     83  HG2 GLU A 177      -6.169   3.216   3.835  1.00  0.00              
ATOM     84  HG3 GLU A 177      -7.694   3.628   4.582  1.00  0.00              
ATOM     85  CD  GLU A 177      -7.913   2.630   2.708  1.00  0.00              
ATOM     86  OE1 GLU A 177      -7.098   2.051   1.786  1.00  0.00              
ATOM     87  OE2 GLU A 177      -9.111   2.387   2.796  1.00  0.00              
ATOM     88  C   GLU A 177      -6.492   6.641   1.063  1.00  0.00              
ATOM     89  O   GLU A 177      -7.357   7.435   1.440  1.00  0.00              
ATOM     90  N   GLY A 178      -5.772   6.833  -0.042  1.00  0.00              
ATOM     91  H   GLY A 178      -4.880   6.331   0.020  1.00  0.00              
ATOM     92  CA  GLY A 178      -5.860   8.074  -0.835  1.00  0.00              
ATOM     93  HA2 GLY A 178      -5.885   7.756  -1.887  1.00  0.00              
ATOM     94  HA3 GLY A 178      -6.815   8.580  -0.656  1.00  0.00              
ATOM     95  C   GLY A 178      -4.725   9.109  -0.691  1.00  0.00              
ATOM     96  O   GLY A 178      -4.747  10.120  -1.394  1.00  0.00              
ATOM     97  N   ASN A 179      -3.721   8.870   0.174  1.00  0.00              
ATOM     98  H   ASN A 179      -3.660   7.889   0.467  1.00  0.00              
ATOM     99  CA  ASN A 179      -2.534   9.734   0.276  1.00  0.00              
ATOM    100  HA  ASN A 179      -2.774  10.769  -0.039  1.00  0.00              
ATOM    101  CB  ASN A 179      -1.906   9.614   1.677  1.00  0.00              
ATOM    102  HB2 ASN A 179      -2.660   9.453   2.473  1.00  0.00              
ATOM    103  HB3 ASN A 179      -1.277   8.700   1.632  1.00  0.00              
ATOM    104  CG  ASN A 179      -0.983  10.771   2.023  1.00  0.00              
ATOM    105  OD1 ASN A 179       0.176  10.813   1.631  1.00  0.00              
ATOM    106  ND2 ASN A 179      -1.477  11.754   2.721  1.00  0.00              
ATOM    107 HD21 ASN A 179      -2.458  11.666   2.999  1.00  0.00              
ATOM    108 HD22 ASN A 179      -0.816  12.512   2.900  1.00  0.00              
ATOM    109  C   ASN A 179      -1.548   9.193  -0.746  1.00  0.00              
ATOM    110  O   ASN A 179      -0.906   8.152  -0.565  1.00  0.00              
ATOM    111  N   LEU A 180      -1.424   9.925  -1.830  1.00  0.00              
ATOM    112  H   LEU A 180      -2.056  10.725  -1.911  1.00  0.00              
ATOM    113  CA  LEU A 180      -0.640   9.435  -2.956  1.00  0.00              
ATOM    114  HA  LEU A 180      -1.065   8.424  -3.028  1.00  0.00              
ATOM    115  CB  LEU A 180      -1.050  10.368  -4.107  1.00  0.00              
ATOM    116  HB2 LEU A 180      -1.211  11.400  -3.714  1.00  0.00              
ATOM    117  HB3 LEU A 180      -0.206  10.417  -4.765  1.00  0.00              
ATOM    118  CG  LEU A 180      -2.242   9.931  -4.966  1.00  0.00              
ATOM    119  HG  LEU A 180      -3.028   9.747  -4.237  1.00  0.00              
ATOM    120  CD1 LEU A 180      -2.693  11.056  -5.911  1.00  0.00              
ATOM    121 HD11 LEU A 180      -2.949  11.981  -5.361  1.00  0.00              
ATOM    122 HD12 LEU A 180      -1.911  11.320  -6.649  1.00  0.00              
ATOM    123 HD13 LEU A 180      -3.596  10.771  -6.482  1.00  0.00              
ATOM    124  CD2 LEU A 180      -1.982   8.651  -5.782  1.00  0.00              
ATOM    125 HD21 LEU A 180      -1.721   7.789  -5.144  1.00  0.00              
ATOM    126 HD22 LEU A 180      -2.874   8.350  -6.364  1.00  0.00              
ATOM    127 HD23 LEU A 180      -1.154   8.781  -6.505  1.00  0.00              
ATOM    128  C   LEU A 180       0.892   9.205  -2.762  1.00  0.00              
ATOM    129  O   LEU A 180       1.484   8.367  -3.436  1.00  0.00              
ATOM    130  N   ALA A 181       1.483   9.917  -1.805  1.00  0.00              
ATOM    131  H   ALA A 181       0.775  10.483  -1.338  1.00  0.00              
ATOM    132  CA  ALA A 181       2.845   9.648  -1.297  1.00  0.00              
ATOM    133  HA  ALA A 181       3.515   9.560  -2.166  1.00  0.00              
ATOM    134  CB  ALA A 181       3.265  10.881  -0.493  1.00  0.00              
ATOM    135  HB1 ALA A 181       3.245  11.795  -1.113  1.00  0.00              
ATOM    136  HB2 ALA A 181       2.605  11.042   0.381  1.00  0.00              
ATOM    137  HB3 ALA A 181       4.295  10.759  -0.112  1.00  0.00              
ATOM    138  C   ALA A 181       2.931   8.310  -0.476  1.00  0.00              
ATOM    139  O   ALA A 181       3.755   7.459  -0.802  1.00  0.00              
ATOM    140  N   CYS A 182       2.059   8.098   0.535  1.00  0.00              
ATOM    141  H   CYS A 182       1.250   8.730   0.491  1.00  0.00              
ATOM    142  CA  CYS A 182       1.928   6.799   1.272  1.00  0.00              
ATOM    143  HA  CYS A 182       2.794   6.550   1.845  1.00  0.00              
ATOM    144  CB  CYS A 182       0.890   7.075   2.353  1.00  0.00              
ATOM    145  HB2 CYS A 182       1.049   8.044   2.837  1.00  0.00              
ATOM    146  HB3 CYS A 182      -0.092   7.147   1.894  1.00  0.00              
ATOM    147  SG  CYS A 182       0.818   5.755   3.572  1.00  0.00              
ATOM    148  C   CYS A 182       1.684   5.554   0.366  1.00  0.00              
ATOM    149  O   CYS A 182       2.363   4.528   0.454  1.00  0.00              
ATOM    150  N   LEU A 183       0.714   5.714  -0.524  1.00  0.00              
ATOM    151  H   LEU A 183       0.210   6.607  -0.441  1.00  0.00              
ATOM    152  CA  LEU A 183       0.402   4.783  -1.618  1.00  0.00              
ATOM    153  HA  LEU A 183       0.114   3.812  -1.187  1.00  0.00              
ATOM    154  CB  LEU A 183      -0.804   5.490  -2.270  1.00  0.00              
ATOM    155  HB2 LEU A 183      -0.518   6.519  -2.539  1.00  0.00              
ATOM    156  HB3 LEU A 183      -0.967   5.052  -3.213  1.00  0.00              
ATOM    157  CG  LEU A 183      -2.185   5.462  -1.614  1.00  0.00              
ATOM    158  HG  LEU A 183      -2.024   5.976  -0.652  1.00  0.00              
ATOM    159  CD1 LEU A 183      -3.172   6.235  -2.505  1.00  0.00              
ATOM    160 HD11 LEU A 183      -3.154   5.874  -3.552  1.00  0.00              
ATOM    161 HD12 LEU A 183      -4.213   6.128  -2.164  1.00  0.00              
ATOM    162 HD13 LEU A 183      -2.946   7.314  -2.537  1.00  0.00              
ATOM    163  CD2 LEU A 183      -2.733   4.034  -1.442  1.00  0.00              
ATOM    164 HD21 LEU A 183      -2.068   3.401  -0.836  1.00  0.00              
ATOM    165 HD22 LEU A 183      -3.718   4.018  -0.942  1.00  0.00              
ATOM    166 HD23 LEU A 183      -2.847   3.514  -2.414  1.00  0.00              
ATOM    167  C   LEU A 183       1.536   4.528  -2.678  1.00  0.00              
ATOM    168  O   LEU A 183       1.607   3.438  -3.243  1.00  0.00              
ATOM    169  N   SER A 184       2.374   5.536  -2.972  1.00  0.00              
ATOM    170  H   SER A 184       2.193   6.344  -2.359  1.00  0.00              
ATOM    171  CA  SER A 184       3.586   5.369  -3.843  1.00  0.00              
ATOM    172  HA  SER A 184       3.288   4.819  -4.759  1.00  0.00              
ATOM    173  CB  SER A 184       4.121   6.746  -4.301  1.00  0.00              
ATOM    174  HB2 SER A 184       4.387   7.377  -3.431  1.00  0.00              
ATOM    175  HB3 SER A 184       5.063   6.614  -4.870  1.00  0.00              
ATOM    176  OG  SER A 184       3.185   7.425  -5.137  1.00  0.00              
ATOM    177  HG  SER A 184       2.446   7.686  -4.562  1.00  0.00              
ATOM    178  C   SER A 184       4.735   4.519  -3.196  1.00  0.00              
ATOM    179  O   SER A 184       5.428   3.787  -3.905  1.00  0.00              
ATOM    180  N   LEU A 185       4.956   4.629  -1.872  1.00  0.00              
ATOM    181  H   LEU A 185       4.267   5.224  -1.388  1.00  0.00              
ATOM    182  CA  LEU A 185       5.909   3.746  -1.127  1.00  0.00              
ATOM    183  HA  LEU A 185       6.844   3.631  -1.679  1.00  0.00              
ATOM    184  CB  LEU A 185       6.205   4.354   0.280  1.00  0.00              
ATOM    185  HB2 LEU A 185       5.327   4.155   0.927  1.00  0.00              
ATOM    186  HB3 LEU A 185       7.027   3.765   0.728  1.00  0.00              
ATOM    187  CG  LEU A 185       6.548   5.857   0.423  1.00  0.00              
ATOM    188  HG  LEU A 185       5.686   6.430   0.030  1.00  0.00              
ATOM    189  CD1 LEU A 185       6.712   6.234   1.904  1.00  0.00              
ATOM    190 HD11 LEU A 185       5.804   5.996   2.488  1.00  0.00              
ATOM    191 HD12 LEU A 185       7.558   5.704   2.380  1.00  0.00              
ATOM    192 HD13 LEU A 185       6.890   7.318   2.029  1.00  0.00              
ATOM    193  CD2 LEU A 185       7.809   6.255  -0.356  1.00  0.00              
ATOM    194 HD21 LEU A 185       7.694   6.056  -1.438  1.00  0.00              
ATOM    195 HD22 LEU A 185       8.028   7.334  -0.257  1.00  0.00              
ATOM    196 HD23 LEU A 185       8.703   5.700  -0.014  1.00  0.00              
ATOM    197  C   LEU A 185       5.478   2.258  -0.965  1.00  0.00              
ATOM    198  O   LEU A 185       6.324   1.362  -0.890  1.00  0.00              
ATOM    199  N   CYS A 186       4.170   2.019  -0.869  1.00  0.00              
ATOM    200  H   CYS A 186       3.679   2.813  -0.455  1.00  0.00              
ATOM    201  CA  CYS A 186       3.620   0.723  -0.550  1.00  0.00              
ATOM    202  HA  CYS A 186       4.129   0.541   0.401  1.00  0.00              
ATOM    203  CB  CYS A 186       2.112   1.056  -0.400  1.00  0.00              
ATOM    204  HB2 CYS A 186       1.697   0.309   0.273  1.00  0.00              
ATOM    205  HB3 CYS A 186       1.818   2.008   0.061  1.00  0.00              
ATOM    206  SG  CYS A 186       1.280   0.928  -1.997  1.00  0.00              
ATOM    207  C   CYS A 186       3.762  -0.568  -1.422  1.00  0.00              
ATOM    208  O   CYS A 186       4.060  -0.589  -2.618  1.00  0.00              
ATOM    209  N   HIS A 187       3.324  -1.641  -0.751  1.00  0.00              
ATOM    210  H   HIS A 187       3.607  -1.553   0.234  1.00  0.00              
ATOM    211  CA  HIS A 187       3.125  -2.978  -1.356  1.00  0.00              
ATOM    212  HA  HIS A 187       3.822  -3.136  -2.195  1.00  0.00              
ATOM    213  CB  HIS A 187       3.402  -4.093  -0.306  1.00  0.00              
ATOM    214  HB2 HIS A 187       2.519  -4.183   0.341  1.00  0.00              
ATOM    215  HB3 HIS A 187       3.459  -5.052  -0.836  1.00  0.00              
ATOM    216  CG  HIS A 187       4.647  -3.984   0.575  1.00  0.00              
ATOM    217  ND1 HIS A 187       5.878  -3.617   0.067  1.00  0.00              
ATOM    218  CD2 HIS A 187       4.598  -3.771   1.965  1.00  0.00              
ATOM    219  CE1 HIS A 187       6.444  -3.127   1.207  1.00  0.00              
ATOM    220  NE2 HIS A 187       5.793  -3.239   2.408  1.00  0.00              
ATOM    221  HD2 HIS A 187       3.688  -3.793   2.549  1.00  0.00              
ATOM    222  HE1 HIS A 187       7.194  -2.364   1.081  1.00  0.00              
ATOM    223  C   HIS A 187       1.631  -3.059  -1.835  1.00  0.00              
ATOM    224  O   HIS A 187       0.698  -2.699  -1.100  1.00  0.00              
ATOM    225  N   ILE A 188       1.375  -3.612  -3.031  1.00  0.00              
ATOM    226  H   ILE A 188       2.207  -3.827  -3.589  1.00  0.00              
ATOM    227  CA  ILE A 188      -0.004  -3.627  -3.623  1.00  0.00              
ATOM    228  HA  ILE A 188      -0.587  -2.766  -3.244  1.00  0.00              
ATOM    229  CB  ILE A 188       0.049  -3.384  -5.172  1.00  0.00              
ATOM    230  HB  ILE A 188       0.667  -4.201  -5.581  1.00  0.00              
ATOM    231  CG2 ILE A 188      -1.355  -3.477  -5.834  1.00  0.00              
ATOM    232 HG21 ILE A 188      -1.831  -4.460  -5.662  1.00  0.00              
ATOM    233 HG22 ILE A 188      -2.050  -2.706  -5.451  1.00  0.00              
ATOM    234 HG23 ILE A 188      -1.309  -3.356  -6.932  1.00  0.00              
ATOM    235  CG1 ILE A 188       0.699  -2.022  -5.577  1.00  0.00              
ATOM    236 HG12 ILE A 188       0.004  -1.184  -5.369  1.00  0.00              
ATOM    237 HG13 ILE A 188       1.575  -1.806  -4.935  1.00  0.00              
ATOM    238  CD1 ILE A 188       1.193  -1.943  -7.034  1.00  0.00              
ATOM    239 HD11 ILE A 188       1.932  -2.734  -7.261  1.00  0.00              
ATOM    240 HD12 ILE A 188       0.370  -2.037  -7.766  1.00  0.00              
ATOM    241 HD13 ILE A 188       1.690  -0.974  -7.234  1.00  0.00              
ATOM    242  C   ILE A 188      -0.711  -4.955  -3.176  1.00  0.00              
ATOM    243  O   ILE A 188      -0.541  -6.030  -3.756  1.00  0.00              
ATOM    244  N   GLU A 189      -1.513  -4.833  -2.108  1.00  0.00              
ATOM    245  H   GLU A 189      -1.382  -3.944  -1.619  1.00  0.00              
ATOM    246  CA  GLU A 189      -2.209  -5.973  -1.454  1.00  0.00              
ATOM    247  HA  GLU A 189      -1.480  -6.793  -1.311  1.00  0.00              
ATOM    248  CB  GLU A 189      -2.665  -5.481  -0.050  1.00  0.00              
ATOM    249  HB2 GLU A 189      -3.336  -4.606  -0.178  1.00  0.00              
ATOM    250  HB3 GLU A 189      -3.311  -6.254   0.413  1.00  0.00              
ATOM    251  CG  GLU A 189      -1.532  -5.145   0.963  1.00  0.00              
ATOM    252  HG2 GLU A 189      -1.014  -6.068   1.283  1.00  0.00              
ATOM    253  HG3 GLU A 189      -0.749  -4.524   0.488  1.00  0.00              
ATOM    254  CD  GLU A 189      -2.013  -4.393   2.198  1.00  0.00              
ATOM    255  OE1 GLU A 189      -2.654  -5.204   3.084  1.00  0.00              
ATOM    256  OE2 GLU A 189      -1.857  -3.186   2.366  1.00  0.00              
ATOM    257  C   GLU A 189      -3.395  -6.583  -2.262  1.00  0.00              
TER     258      GLU A 189 
END   
