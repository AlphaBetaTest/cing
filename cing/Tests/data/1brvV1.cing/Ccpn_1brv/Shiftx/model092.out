# Entries marked with a * may have inaccurate shift predictions.
# Entries marked with a value -666 should be igonored
# Output format adjusted by GV 11 September 2007
  171   V  N      114.0562
  171   V  CA      60.5985
  171   V  CB      32.3148
  171   V  C      171.9493
  171   V  H        7.8976
  171   V  HA       4.0371
  171   V  HB       2.0142
  171   V  HG11     0.9220
  171   V  HG12     0.9520
  171   V  HG13     0.6180
  171   V  HG21     0.6863
  171   V  HG22     0.9804
  171   V  HG23     0.9308
  172   P  CA      60.2087
  172   P  CB      33.2608
  172   P  C      176.5575
  172   P  HA       4.9550
  172   P  HB2      2.3710
  172   P  HB3      2.1845
  172   P  HD2      3.6956
  172   P  HD3      3.7660
  172   P  HG2      1.9808
  172   P  HG3      2.0595
  173   C  N      121.3186
  173   C  CA      59.9052
  173   C  CB      40.8197
  173   C  C      177.3581
  173   C  H        8.2563
  173   C  HA       4.2811
  173   C  HB2      3.3050
  173   C  HB3      3.1788
  174   S  N      114.9364
  174   S  CA      60.7982
  174   S  CB      62.3993
  174   S  C      175.5805
  174   S  H        8.1758
  174   S  HA       4.1602
  174   S  HB2      4.0529
  174   S  HB3      4.0154
  175   T  N      111.5269
  175   T  CA      62.6745
  175   T  CB      69.9712
  175   T  C      175.6872
  175   T  H        7.6571
  175   T  HA       4.4294
  175   T  HB       4.2392
  175   T  HG21     1.2113
  175   T  HG22     1.2121
  175   T  HG23     1.2122
  176   C  N      119.4497
  176   C  CA      58.4337
  176   C  CB      40.8583
  176   C  C      173.7162
  176   C  H        8.1415
  176   C  HA       4.3439
  176   C  HB2      2.9785
  176   C  HB3      3.2038
  177   E  N      116.4460
  177   E  CA      57.5692
  177   E  CB      27.0298
  177   E  C      175.4790
  177   E  H        8.9269
  177   E  HA       3.6663
  177   E  HB2      2.1121
  177   E  HB3      1.9739
  177   E  HG2      2.3246
  177   E  HG3      2.3754
  178   G  N      104.1707
  178   G  CA      45.1989
  178   G  C      173.5549
  178   G  H        8.3402
  178   G  HA    -666.0000
  178   G  HA2      3.9577
  178   G  HA3      3.9573
  179   N  N      119.2354
  179   N  CA      52.4232
  179   N  CB      39.3107
  179   N  C      175.2409
  179   N  H        7.9486
  179   N  HA       4.5952
  179   N  HB2      2.7959
  179   N  HB3      2.7279
  179   N  HD21     7.1331
  179   N  HD22     6.5548
  180   L  N      129.1676
  180   L  CA      57.2762
  180   L  CB      41.7220
  180   L  C      179.4289
  180   L  H        8.4514
  180   L  HA       3.9659
  180   L  HB2      1.7228
  180   L  HB3      1.7007
  180   L  HD11     0.9178
  180   L  HD12     0.9178
  180   L  HD13     0.9179
  180   L  HD21     0.2252
  180   L  HD22     1.9529
  180   L  HD23     0.8669
  180   L  HG       1.7182
  181   A  N      120.7072
  181   A  CA      55.2170
  181   A  CB      18.0225
  181   A  C      179.6749
  181   A  H        7.9034
  181   A  HA       3.9022
  181   A  HB1      1.3853
  181   A  HB2      1.3596
  181   A  HB3      1.3319
  182   C  N      116.4217
  182   C  CA      60.1409
  182   C  CB      43.4653
  182   C  C      175.9285
  182   C  H        7.9619
  182   C  HA       4.0887
  182   C  HB2      3.0070
  182   C  HB3      3.2302
  183   L  N      118.9909
  183   L  CA      57.9192
  183   L  CB      41.1923
  183   L  C      179.4771
  183   L  H        7.9252
  183   L  HA       3.9090
  183   L  HB2      1.8662
  183   L  HB3      1.7119
  183   L  HD11     0.9153
  183   L  HD12     0.9892
  183   L  HD13     0.9011
  183   L  HD21     0.4242
  183   L  HD22     2.3492
  183   L  HD23     0.8634
  183   L  HG       1.6805
  184   S  N      114.4602
  184   S  CA      61.4654
  184   S  CB      62.8162
  184   S  C      176.3646
  184   S  H        7.8597
  184   S  HA       4.2760
  184   S  HB2      4.0090
  184   S  HB3      3.9153
  185   L  N      120.8995
  185   L  CA      56.7363
  185   L  CB      41.5375
  185   L  C      174.9751
  185   L  H        7.6368
  185   L  HA       4.1640
  185   L  HB2      1.5757
  185   L  HB3      1.7054
  185   L  HD11     0.9084
  185   L  HD12     0.9118
  185   L  HD13     0.9130
  185   L  HD21     0.3340
  185   L  HD22     2.6485
  185   L  HD23     0.8637
  185   L  HG       1.7944
  186   C  N      111.4884
  186   C  CA      55.2519
  186   C  CB      41.1754
  186   C  C      172.9756
  186   C  H        7.1151
  186   C  HA       5.2193
  186   C  HB2      3.1070
  186   C  HB3      3.2619
  187   H  N      115.8339
  187   H  CA      54.9739
  187   H  CB      30.2396
  187   H  C      175.1926
  187   H  H        8.1878
  187   H  HA       4.7904
  187   H  HB2      3.0656
  187   H  HB3      3.2660
  187   H  HD2      5.6052
  187   H  HE1      6.7416
  188   I  N      126.4658
  188   I  CA      60.6236
  188   I  CB      36.9720
  188   I  C      176.4601
  188   I  H        8.4484
  188   I  HA       4.1991
  188   I  HB       1.8789
  188   I  HD11     0.8653
  188   I  HD12     0.8624
  188   I  HD13     0.8611
  188   I  HG12     1.2030
  188   I  HG13     0.5058
  188   I  HG21     0.9153
  188   I  HG22     0.9127
  188   I  HG23     0.9135
 *189   E  N      121.9912
 *189   E  CA      55.9417
 *189   E  CB      30.3082
 *189   E  C      173.8715
 *189   E  H        8.4722
 *189   E  HA       4.1395
 *189   E  HB2      1.7400
 *189   E  HB3      1.7717
 *189   E  HG2      2.1488
 *189   E  HG3      2.5567
